### ISSUE TITLE
Large size difference in matfile versions

### ISSUE THREAD

**Issue by gehrhorn** [gehrhorn/baseball-stats#12]
See [here](http://www.mathworks.com/help/matlab/import_export/mat-file-versions.html), [here](http://www.mathworks.com/help/matlab/matlab_env/preferences.html#brjh09y-7), and [here](http://stackoverflow.com/questions/4950630/matlab-differences-between-mat-versions). Testing in  28ec58b

**Comment by gehrhorn** [comment]
28ec58

**Comment by gehrhorn** [comment]
``` zsh
~/MATLAB-Drive/baseball <matfile-versions> Χ ll *.mat                     george at alan
-rw-r--r--  1 george  staff   7.3M Jan  6 19:12 baseballstats7.mat
-rw-r--r--  1 george  staff   106M Jan  6 19:12 baseballstats73.mat
```

Fixed in 28ec58b
